include/revng/Model/CallEdge.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Model/AbstractFunctionEdge.h"
+#include "revng/Model/VerifyHelper.h"
+
+/* MODEL-YAML
+name: CallEdge
+doc: A CFG edge to represent function calls (direct, indirect and tail calls)
+type: struct
+inherits: FunctionEdge
```

Comment:
This document has two `inherits` entries

Reply:
And python YAML parser apparently has no issue with it, I'm mad